src/App.jsx

```diff
@@ -17,22 +17,80 @@ const Films = React.lazy(() => import('./components/films'));
 
 const App = () => {
   const [darkTheme, setDarkTheme] = useState(false);
+  const [films, setFilms] = useState([]);
+  const [userFilms, setUserFilms] = useState([]);
+  const [nextPage, setNextPage] = useState(1);
+
+  useEffect(() => {
+    setUserFilms(JSON.parse(localStorage.getItem('userFilms')));
```

Comment:
if nothing exists in localStorage, you can set it with logical operator so to get rid of redundant if/else in next useEffect `setUserFilms(JSON.parse(localStorage.getItem('userFilms') || []));`